### DISCUSSION TITLE
Add flags for linking external C library (lame) on windows

### DISCUSSION THREAD

**Discussion by olup** [wailsapp/wails#1585] [Q&A] [answered]
On Windows (in a GH action) I can't really find the way to build my wails app linking the "lame" C library. Usually, I just add env variables before building, pointing to a directory of the repo that contains the libs and includes for windows. But blindly setting those before building with wails CLI is not working, and I expect this is because the CLI build by passing its own flags.

For reference this is what I do in my CI (lunii-admin being my repo's name)

  Set-Item -Path Env:CGO_LDFLAGS -Value "-LD:/a/lunii-admin/lunii-admin/bindings/windows/lib"
  Set-Item -Path Env:CGO_CFLAGS -Value "-ID:/a/lunii-admin/lunii-admin/bindings/windows/include"
Any guidance on this use case ?

(ps: it builds fine on macos amd and linux, but in those platforms lame is located in the sysroot)

Cheers.

**Comment by olup** [comment, marked as answer]
I was using a helper action. Going back to pure "wails build" command worked like a charm.

**Reply by wlwatkins** [reply]
I'm facing the same issue for zmq. could you elaborate how you fixed it. it's the first time i am needing to include c files in a go project, so I'm not even sure what to do with 

```
$env:CGO_CFLAGS='-ID:/dev/vcpkg/installed/x64-windows/include'
$env:CGO_LDFLAGS='-LD:/dev/vcpkg/installed/x64-windows/lib -l:libzmq-mt-4_3_4.lib'
```

I got the *.h and *.dll files in my repo, how do i tell wails to use them when calling `wails dev`? thanks

**Reply by leaanthony** [reply]
It *should* pick those up. Try `wails build -v 2` and check the environment variables 👍 Also, we need a guide on this so if anyone would like to volunteer.... 😉

**Reply by wlwatkins** [reply]
ok, i managed it to work but now i'm having issues with the compiler. installing libzmq on windows has so far been impossible for me. but that's an issue for another repo
